unit_test/sum.js

```diff
@@ -0,0 +1,7 @@
+
+function sum(a, b) {
```

Comment:
Is this file intentional?

Reply:
Yes, they are my files, to test how Jest works. Any suggestion on where to place them? If not, I will just move them to a separate folder then.

Reply:
Sounds good! Will do.